documentation/modules/proc-iot-management-token.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// assembly-iot-creating-device.adoc
+
+[id='proc-iot-management-token-{context}']
+= Getting an authentication token
+
+In order to access the Device Management API, you will need a token for authenticating yourself
+against the management API.
+
+Access to devices of an IoT tenant is mapped by the device registry based on access to
+the `IoTProject` resource. If an account has *read* access to the `IoTProject`, the this
+account can also execute *read* operations on the device registry for this IoT tenant.
+
+The token has to be presented to the API as a *bearer token*. This is done by adding
+an HTTP header value: `Authorization: Bearer <token>`. For more information,
+see https://tools.ietf.org/html/rfc6750[RFC 6750].
+
+In the following examples, the variable `${TOKEN}` will be used as a placeholder.
+Be sure replace this variable with the actual token.
+
+[id='proc-iot-management-token-from-user{context}']
+== From a user
+
+If you want to use the token of the current {KubePlatform}, you can extract the
+token by executing the following command:
+
+.Prerequisites
+
+* Be logged in to your {KubePlatform} instance with a user that supports tokens
+
+.Procedure
+
+. Extract the token for the current user
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+ifeval::["{cmdcli}" == "oc"]
+oc whoami -t
+endif::[]
+ifeval::["{cmdcli}" == "kubectl"]
+kubectl config view -o json| jq -r '.users[] as $users | ."current-context" as $ctx | .contexts[] | select (.name==$ctx).context.user as $user | $users | select (.name==$user).user.token'
+endif::[]
+----
+
+NOTE: User tokens mostley have a limited lifetime. So it may be necessary
+to re-acquire the token after it has expired.
+
+[id='proc-iot-management-token-from-service-account{context}']
+== From a service account
+
+In order to create a new service account and extract the token you need to perform
+the following steps:
+
+.Prerequisites
+
+* Be logged in to your {KubePlatform} instance with permissions to create new service accounts, roles and role bindings.
+
+.Procedure
+
+. Create a new service account
++
+[source,yaml,options="nowrap"]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: my-device-manager-account # <1>
+----
+<1> The name of the service account
+
+. Create a new role, allowing access to the `IoTProject`
++
+[source,yaml,options="nowrap"]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: device-manager-role # <1>
+rules: # <2>
+- apiGroups: ["iot.enmasse.io"]
+  resources: ["iotprojects"]
+  verbs: ["create", "update", "get", "list", "delete"]
+----
+<1> The name of the role
+<2> The access rules, which must grant CRUD access to the `IoTProject`.
++
+This example grants access to all `IoTProjects`s in a namespace. It is possible
```

Comment:
```suggestion
This example grants access to all IoT projects in a namespace.
```

Reply:
`IoTProject` is actually the name of the resource in Kubernetes. Should I still replace it?